src/lucky_web/log_handler.cr

```diff
@@ -12,12 +16,18 @@ class LuckyWeb::LogHandler
     elapsed = Time.now - time
     elapsed_text = elapsed_text(elapsed)
 
-    @io.puts "#{context.request.method} #{context.response.status_code} #{context.request.resource.colorize(:green)} (#{elapsed_text})"
+    timestamp = if settings.show_timestamps
```

Comment:
This could be it's own private method I think. `private def timestamp` and you could call `Time.new` in it so you don't have to pass anything to it

Reply:
wouldn't that record timestamp of request end instead of start?

Reply:
Yes you're right. You should pass the time so it records when it started. Good catch!